Containers/nextcloud/cron.sh

```diff
@@ -5,3 +5,6 @@ while true; do
     php -f /var/www/html/cron.php &
     sleep 5m
 done
+
+# Wait for last command to end
+wait $!
```

Comment:
@Zoey2936 I have no clue if this actually works...

Reply:
https://stackoverflow.com/a/3004814 I meant that

Reply:
I see and where would I add this inside this script?

Reply:
see like I did it here in the last 2 lines https://github.com/ZoeyVid/nginx-proxy-manager/blob/develop/rootfs/bin/launch.sh and this: https://stackoverflow.com/a/68931259

Reply:
not sure if it makes a difference to use 
command1 &
command2 &
wait

or

command1 & 
command2
wait